scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroupInternals.scala

```diff
@@ -69,11 +83,18 @@ private[peergroup] object DynamicTLSPeerGroupInternals {
   class MessageNotifier[M](
       messageQueue: ChannelAwareQueue[ChannelEvent[M]],
       codec: Codec[M],
-      eventLoop: EventLoop,
-      channelConfig: ChannelConfig
+      eventLoop: EventLoop
   ) extends ChannelInboundHandlerAdapter
       with StrictLogging {
 
+    private def idleEventToChannelEvent(idleStateEvent: IdleStateEvent): ChannelIdle = {
+      idleStateEvent.state() match {
+        case IdleState.READER_IDLE => ChannelIdle(ReaderIdle, idleStateEvent.isFirst)
+        case IdleState.WRITER_IDLE => ChannelIdle(WriterIdle, idleStateEvent.isFirst)
+        case IdleState.ALL_IDLE => ChannelIdle(AllIdle, idleStateEvent.isFirst)
+      }
```

Comment:
Some comments would be helpful on the `Channeldle` class about what to do with `isFirst` 
* When it goes `READER -> ALL -> READER`, is the second `READER` a first? 
* If a connection idles, then there's activity, then idles again, is the second one a first? 
* Are idle events raised just once per idleness? Or once per timeout, if it keeps being idle?

Reply:
sure thing I will add those docs :+1: